src/utils/format.ts

```diff
@@ -56,17 +56,17 @@ export function sanitizeNegativeAndMakeMultipleOf(value?: string | null, default
     : defaultValue
 }
 
-export function validatePositive(value: string): true | string {
-  return Number(value) > 0 || 'Invalid amount'
+export function validatePositive(value: string, constraint = 0): true | string {
+  return Number(value) > constraint || 'Invalid amount'
 }
 export const validInputPattern = new RegExp(/^\d+\.?\d*$/) // allows leading and trailing zeros
 export const leadingAndTrailingZeros = new RegExp(/(^0*(?=\d)|\.0*$)/, 'g') // removes leading zeros and trailing '.' followed by zeros
 export const trailingZerosAfterDot = new RegExp(/(.*\.\d+?)0*$/) // selects valid input without leading zeros after '.'
 
-export const formatValidity = (validTime: string | number): string =>
-  +validTime == 0
-    ? 'Unlimited'
-    : +validTime < 0
+export const formatValidity = (validTime: string | number, zeroValueText: string, constraint = 0): string =>
```

Comment:
I think you are making this `formatValidity` more general. So you cannot call it `formatValidity` any more.

I'm not good with names, maybe `formatTimeMinutes`